backend/dataall/base/aws/sts.py

```diff
@@ -364,6 +361,7 @@ def generate_console_url(credentials, session_duration=None, region='eu-west-1',
 
     @staticmethod
     def is_assumable_pivot_role(accountid):
+        # TODO: region
```

Comment:
need to pass region here as well right?

Reply:
My bad, I needed a final review of my TODOs. Fixed! Yes, `lf_share_manager` is the only place where it is called (it is a function used for central catalog feature)